app/error_messages.py

```diff
@@ -40,3 +40,4 @@
     "Variants have more than one question type for block."
 )
 PREVIEW_WITHOUT_INTRODUCTION_BLOCK = "No introduction block found. Introduction block is mandatory when using the preview questions feature."
+LIST_REFERENCED_BEFORE_ADDED = "List referenced as source before it has been created."
```

Comment:
Really minor but would make it consistent and use created rather than added as the var name too.